privacyidea/lib/event.py

```diff
@@ -159,17 +159,20 @@ def get_handler_object(handlername):
     from privacyidea.lib.eventhandler.federationhandler import \
         FederationEventHandler
     from privacyidea.lib.eventhandler.counterhandler import CounterEventHandler
+    from privacyidea.lib.eventhandler.responsemangler import ResponseManglerHandler
     h_obj = None
     if handlername == "UserNotification":
         h_obj = UserNotificationEventHandler()
-    if handlername == "Token":
+    elif handlername == "Token":
         h_obj = TokenEventHandler()
-    if handlername == "Script":
+    elif handlername == "Script":
         h_obj = ScriptEventHandler()
-    if handlername == "Federation":
+    elif handlername == "Federation":
         h_obj = FederationEventHandler()
-    if handlername == "Counter":
+    elif handlername == "Counter":
         h_obj = CounterEventHandler()
+    elif handlername == "ResponseMangler":
+        h_obj = ResponseManglerHandler()
```

Comment:
I think we should keep the naming scheme and call it ``ResponseManglerEventHandler()``. That way we can later add arbitrary handler more easy by using the ``handlername + 'EventHandler'`` object.

Reply:
Done in 070a10f9
Please check and resolve the conversation.